package.json

```diff
@@ -21,8 +21,12 @@
     "atom": ">=0.174.0"
   },
   "dependencies": {
-    "atom-linter": "^3.0.0"
+    "atom-linter": "^3.0.0",
+    "atom-package-deps": "^3.0.0"
```

Comment:
It's at least my opinion that one should strive to specify the lowest possible version of compatibility in dependencies. Since it doesn't actually **need** the latest version.
But I'll change it if you think that'd be better.